tests/test_individual.py

```diff
@@ -13,16 +13,16 @@ def test_evaluate(self):
         ind = Individual(chromosome=(1, 2))
         ind.evaluate(sum)
 
-        def eval_func(chromosome):
+        def eval_func1(chromosome):
```

Comment:
Is there a technical reason that prevents us from using two evaluation functions with the same name?

Reply:
The linter in my IDE gave me grief over it.